jupyterhub/spawner.py

```diff
@@ -255,6 +320,17 @@ def get_env(self):
                 env[key] = value
 
         env['JPY_API_TOKEN'] = self.api_token
+
+        # Put in limit and guarantee info if they exist.
```

Comment:
Discussed in person: these are for humans, not the Spawner, so these should stay and we can just add a comment clarifying that.